### PR TITLE
Synchronize arming

### PR DESCRIPTION
**Pull request by RobertSchaffer1**
@JunAishima @vshekar Changes to parallelize the arming and governor moves, requires https://github.com/NSLS-II-AMX/mxtools/pull/33

### CODE REVIEW

File: daq_macros.py [hunk 1]
```diff
@@ -1854,17 +1851,23 @@ def snakeRasterBluesky(rasterReqID, grain=""):
...
+    arm_status = raster_flyer.detector_arm(angle_start=omega, img_width=img_width_per_cell, total_num_images=totalImages, exposure_period_per_image=exptimePerCell, file_prefix=rasterFilePrefix,
                        data_directory_name=data_directory_name, file_number_start=file_number_start, x_beam=xbeam, y_beam=ybeam, wavelength=wave, det_distance_m=detDist,
                        num_images_per_file=numsteps)
+    govStatus = gov_lib.setGovRobot(gov_robot, "DA")
+    if govStatus.exception():
+      logger.error(f"Problem during start-of-raster governor move, aborting! exception: {govStatus.exception()}")
+      arm_status.wait()
+      raster_flyer.detector.cam.acquire.put(0)
+      return
+    arm_status.wait()
```
Comment (JunAishima) [inline daq_macros.py]: every time we have an arm_status.wait(), can we output the time difference from when arming is started?

**Review by JunAishima — CHANGES_REQUESTED** [review]
logging for arm+governor time would be very nice to have here

File: daq_macros.py (hunk 1 above)
Comment (RobertSchaffer1) [reply daq_macros.py]: Sure, I suppose.

Comment (RobertSchaffer1) [reply daq_macros.py]: Done.

**Review by JunAishima — APPROVED** [review]

**Pull request merged**